docs/src/assets/scss/components/social-icons.scss

```diff
@@ -7,10 +7,11 @@
         padding: 0;
         margin-left: -1rem;
         margin-inline-start: -1rem;
+        display: flex;
+        justify-content: center;
```

Comment:
```suggestion
        display: inline-flex;
```
If we use `display: flex;` with `justify-content: center;` the items are not aligned towards the left in the desktop view.

  We can use `display: inline-flex;` it is working on screen sizes.

Reply:
Hmmm, they look aligned to the left to me.
![Screenshot 2023-01-20 at 10-39-52 Documentation - ESLint - Pluggable JavaScript Linter](https://user-images.githubusercontent.com/38546/213779910-6d6c4ccc-c81c-4b75-90f8-ccaf58ccd4e8.png)

Reply:
| Preview | Current (eslint.org) |
|---------|---------|
| ![Screenshot 2023-01-21 at 12 33 40 AM](https://user-images.githubusercontent.com/30730124/213784688-64271e80-92e4-4c5d-80a9-b982c6119758.png) | ![Screenshot 2023-01-21 at 12 32 56 AM](https://user-images.githubusercontent.com/30730124/213784609-dbb6a85f-1f19-455c-b2e7-fa8e43d3b80b.png)